notebooks/text-classification_multilabel.ipynb

```diff
@@ -0,0 +1,336 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "552ec552",
+   "metadata": {},
+   "source": [
+    "# SetFit for Text Classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7250e1a6",
+   "metadata": {},
+   "source": [
+    "Welcome to SetFit for Text Classification! In this short notebook exercise we'll go into just how easy it is to do few-shot classification with SetFit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5bf4ce45",
+   "metadata": {},
+   "source": [
+    "To start, we install SetFit which will install all of the dependencies you will need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "712b96e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install setfit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78e8c41a",
+   "metadata": {},
+   "source": [
+    "We then import relevant functions (data loading and loss function) and our handy SetFitModel and SetFitTrainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e44b7069-27b0-49ea-bc27-c44f94a98e2f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/michelepangrazzi/workspace/setfit/.venv/lib/python3.8/site-packages/tqdm/auto.py:22: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "from datasets import load_dataset\n",
+    "from sentence_transformers.losses import CosineSimilarityLoss\n",
+    "\n",
+    "from setfit import SetFitModel, SetFitTrainer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cbd3e642",
+   "metadata": {},
+   "source": [
+    "We start by loading a dataset to work with. In this example we'll load the \"emotion\" dataset already on the HF hub."
```

Comment:
The `emotion` dataset isn't a true multilabel dataset, so what do you think about trying something like `lex_glue`?